Makefile

```diff
@@ -187,7 +187,7 @@ elasticsearch-mappings:
 .PHONY: build-examples
 build-examples:
 	esc -pkg frontend -o examples/hotrod/services/frontend/gen_assets.go  -prefix examples/hotrod/services/frontend/web_assets examples/hotrod/services/frontend/web_assets
-	CGO_ENABLED=0 installsuffix=cgo go build -o ./examples/hotrod/hotrod-$(GOOS) ./examples/hotrod/main.go
+	installsuffix=cgo go build -o ./examples/hotrod/hotrod-$(GOOS) ./examples/hotrod/main.go
```

Comment:
`installsuffix=cgo` can also go away.

cf. https://medium.com/@daviddavid533/cgo-enabled-0-and-installsuffix-cgo-are-no-longer-required-since-go-1-10-127d8c265cb1

Reply:
Done